setup.cfg

```diff
@@ -5,7 +5,7 @@ max-line-length = 100
 
 [tool:pytest]
 addopts = -rsxX --tb=native
-norecursedirs = .git docs scripts config docker __pycache__
+norecursedirs = __pycache__
```

Comment:
I don't understand why this change happened. Why remove all these directories from the `norecursedirs`?